web/client/utils/mapinfo/wmts.js

```diff
@@ -78,5 +82,25 @@ export default {
             },
             url: getLayerUrl(layer).replace(/[?].*$/g, '')
         };
-    }
+    },
+    getIdentifyFlow: (layer, basePath, params) =>
+        Observable.defer(() => axios.get(basePath, { params }))
+            .catch((e) => {
```

Comment:
Why would I add exception handler without even trying it?
It can be checked using #/geostory/41714
Here it is, thrown upon click outside of the area where data is available, status code is 400:
![image](https://user-images.githubusercontent.com/113525996/197767265-227b494c-98e4-4366-bbc4-dc017da8408d.png)

Reply:
I guess the issue is that response code is not 200, so it's not getting handled by "interceptOGCError".

Reply:
Sorry, I didn't want to accuse of course  :)  
When I said "I'm afraid you added the code but you didn't tried it." I meant that you committed in error.
It is strange, usually they respond with 200. I had not data to try it, so I wasn't able to verify myself. 
I will try with #/geostory/41714, thank you for clarifying

Reply:
Ok I tested it. Sorry my fault.
I did an error reading the code. Also returning exception I think it is ok. You can revert the changes.
Unit test should be added.